### PR TITLE
Fix if not set GAPPS_VARIANT

### PR DESCRIPTION
**Pull request by lrinQVQ**

### CODE REVIEW

**Comment by Blystad** [comment]
I'm not sure I follow the motivation for this PR. Can you explain more what you're trying to accomplish?

File: README.md [hunk 1]
```diff
@@ -76,7 +76,9 @@ The OpenGApps AOSP based build system is placed in `vendor/google/build` and inc
 in `build/core/main.mk`:
 ```
 # Include the google-specific config
+ifneq ($(GAPPS_VARIANT),)
```
Comment (Blystad) [inline README.md]: This is a reference to existing code in AOSP...not something that should be modified.

Comment (lrinQVQ) [inline README.md]: Ok...I just hope Opengapps to better support Other rom... such as:Cyanogenmod,Thanks

Comment (lrinQVQ) [inline README.md]: If any device.mk hava set GAPPS_VARIANT, but another device.mk not set GAPPS_VARIANT,In Cyanogenmod,If I common lunch cm_device-userdbug && make -j9,shell will print "error GAPPS_VARIANT was not found. Use of one of pico,nano,micro,mini,full,stock,super)"

**Comment by Blystad** [comment]
Ok, that explains it a little bit better.

I don't think this is something the opengapps aosp_build project should be responsible for handling _per se_. If Opengapps is included in the manifest file then you must provide a GAPPS_VARIANT value, this is by design.

As explained in the README aosp_build is mostly for developers and people creating their own builds. CyanogenMod couldn't ship with `aosp_build` artifacts pre-installed, they have to ship the ROM code and the Gapps as seperate ZIP files to adhere to Googles cease and desist letter.

**Comment by lrinQVQ** [comment]
OK~Thanks

**Pull request closed without merging**